services/ui-src/src/components/banners/PreviewBanner.tsx

```diff
@@ -0,0 +1,23 @@
+import { useFormContext } from "react-hook-form";
+// components
+import { Banner } from "components";
+
+export const PreviewBanner = ({ ...props }: Props) => {
+  // get the form context
+  const form = useFormContext();
+
+  // set banner preview data
+  const formData = form.getValues();
+  const bannerData = {
+    title: formData?.["bannerTitle"]?.["answer"] || "New banner title",
+    description:
+      formData?.["bannerDescription"]?.["answer"] || "New banner description",
+    link: formData?.["bannerLink"]?.["answer"] || "",
+  };
+
+  return <Banner bannerData={bannerData} {...props} />;
+};
+
+interface Props {
+  [key: string]: any;
```

Comment:
If there are no properties in the props, would we just omit the argument on line 5 entirely? I think that works, right? 🤔 

```suggestion
```